_es/tour/tour-of-scala.md

```diff
@@ -31,16 +31,25 @@ Scala cuenta con un expresivo sistema de tipado que fuerza estáticamente las ab
 * [métodos polimórficos](polymorphic-methods.html)
 
-El [mecanismo de inferencia de tipos locales](type-inference.html) se encarga de que el usuario no tenga que anotar el programa con información redundante de tipado. Combinadas, estas características proveen una base poderosa para el reuso seguro de abstracciones de programación y para la extensión segura (en cuanto a tipos) de software.
+El [mecanismo de inferencia de tipos locales](type-inference.html) se encarga de que el usuario no tenga que anotar el programa con información redundante de tipado. Combinadas, estas características proveen una base poderosa para la reutilización segura de abstracciones de programación y para la extensión segura (en cuanto a tipos) de software.
 
 ## Scala es extensible ##
 
-En la práctica el desarrollo de aplicaciones específicas para un dominio generalmente requiere de "Lenguajes de dominio específico" (DSL). Scala provee una única combinación de mecanismos del lenguaje que simplifican la creación de construcciones propias del lenguaje en forma de librerías:
+En la práctica, el desarrollo de aplicaciones específicas para un dominio generalmente requiere de "Lenguajes de dominio específico" (DSL). Scala provee una única combinación de mecanismos del lenguaje que simplifican la creación de construcciones propias del lenguaje en forma de bibliotecas:
 * cualquier método puede ser usado como un operador de [infijo o postfijo](operators.html)
 * [las closures son construidas automáticamente dependiendo del tipo esperado](automatic-closures.html) (tipos objetivo).
 
 El uso conjunto de ambas características facilita la definición de nuevas sentencias sin tener que extender la sintaxis y sin usar facciones de meta-programación como tipo macros.
 
-Scala está diseñado para interoperar bien con el popular Entorno de ejecución de Java 2 (JRE). En particular, la interacción con el lenguaje orientado a objetos Java es muy sencillo. Scala tiene el mismo esquema de compilación (compilación separada, carga de clases dinámica) que java y permite acceder a las miles de librerías de gran calidad existentes.
+## Scala interopera
+
+Scala está diseñado para interoperar correctamente con la popular Java Runtime Environment (JRE).
+En particular, es posible la misma interacción con el lenguaje de programación Java.
+Nuevas características de Java como SAMs, [lambdas](higher-order-functions.html), [annotations](annotations.html), y [generics](generic-classes.html) tienen sus análogos en Scala.
+
+Aquellas características de Scala que no tienen analogías en Java, como por ejemplo [default](default-parameter-values.html) y [named parameters](named-arguments.html), compilan de una forma tan similar a Java como es razonablemente posible. 
+Scala tiene el mismo modelo de compilación (compilación separada, carga de clases dinámica) que Java y permite acceder a miles de bibliotecas de alta calidad ya existentes.
+
+## ¡Disfruta el tour!
```

Comment:
Very good change, the paragraph was greatly enriched.

Reply:
And the links? parámetros con valores por defecto, parámetros nombrados, etcétera

Reply:
Good point, I forgot to translate the links